src/python/Connection.py

```diff
@@ -115,3 +116,64 @@ def setValues(self, values):
             return
         elif reply_node.type == Reply_pb2.Reply.Type.ERROR:
             raise ConnectionError(reply_node.error_reply.message_data)
+
+    def Subscribe(self, uri, callback=None):
+        """Subscribes to the specified uri in the network table
+
+        uri - a string 
```

Comment:
Can you remove the whitespace at the end of the following lines:
src/python/Connection.py:123:23: W291 trailing whitespace
src/python/Connection.py:124:87: W291 trailing whitespace
src/python/Connection.py:137:1: W293 blank line contains whitespace
src/python/Connection.py:140:19: W291 trailing whitespace
src/python/Connection.py:143:1: W293 blank line contains whitespace
src/python/Connection.py:147:106: W291 trailing whitespace
src/python/Connection.py:156:1: W293 blank line contains whitespace
src/python/Connection.py:161:53: W291 trailing whitespace